[DOC] Make public two-week policy for closing issues

Serabe opened the pull request:
When triaging a team, we use the _Needs Submitter Response_ to mark
issues that cannot be solved without some extra information from the
original submitter. If the submitter does not provide any extra
information after seven days, the team gently reminds the original
submitter. If another week passes, the issue might be closed.

Serabe committed:
[DOC] Make public two-week policy for closing issues

When triaging a team, we use the _Needs Submitter Response_ to mark
issues that cannot be solved without some extra information from the
original submitter. If the submitter does not provide any extra
information after seven days, the team gently reminds the original
submitter. If another week passes, the issue might be closed.

Serabe commented:
@locks, @pixelhandler: a draft of what we talked about today!

locks commented:
I think you should introduce a triaging policy header so we can list our policies, starting with this one, and the lifecycle of an issue. The copy is 👍 , so it's a good start.

rwjblue commented:
It seems great to document and codify the policies we follow here. I'll try to get confirmation from the rest of the core team on this during the F2F meeting this weekend.

stefanpenner commented:
👍

Serabe commented:
Thank you for your comments and so sorry for the long delay. I've added the header, as @locks suggested. I've also added what we currently do when the issue submitted is actually a feature request.

I think these two are the main policies we are following right now and are a good starting point for this public policy.

Thank you again for taking your time to review this!

tomdale commented:
LGTM

rwjblue commented:
Discussed with core team today (sorry for prolonged delay), we are 👍. Might tweak the process/verbiage a bit over time, but this is good to go...
